schema.json

```diff
@@ -1,15 +0,0 @@
-{
-  "users": [{
-    "id": "",
-    "email": "",
-  }],
-  "posts": [{
-    "id": "",
-    "author": {
-      "name": "",
-
```

Comment:
Please remove the extra new line.

Reply:
@pchalubinska is right, I can't remove this as I deleted the whole `schema.json` file already.

Reply:
My mistake then. Will approve. Thanks @zsoltime @pchalubinska .